proto/minder/v1/minder.proto

```diff
@@ -1987,12 +2028,57 @@ message RoleAssignment {
     string role = 1;
```

Comment:
I think that one thing that we are missing is a displayName for the role, because we should display "Read-only" instead of "read_only" on the UI

(and the same applies not only for invitations but also for the endpoint that lists existing role assignments)

Reply:
Look at `ListRoles` / `ListRolesResponse` and the contents of the `Role` message there.

I'm assuming that you're requesting that we duplicate that data into each RoleAssignment, but I'm going to push back on that, at least for right now -- from an API consumer perspective, it raises the question as to whether roles with the same name can have different descriptions.

Reply:
in that response description is akin to displayName? In other words it's a short name that we can use to display in the UI? (I assumed that description is potentially some multi-line text, or at least something too long to use as a name)

Reply:
I'm fine with the idea of just combining 2 responses, that is not a problem for us, but we should be aware that such design would require all clients to duplicate that logic. If we think about all of Minder as a complex system (GUI, CLI and BE) then often a good tradeoff might be to deal with that complexity in the BE side. (I don't have deep knowledge of the CLI codebase of course, I'm just assuming that the client data fetching logic must be very similar to what we have in the web gui)